service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -99,6 +104,48 @@ void setup() {
             when(feeService.getFeesDataForC2(WITHOUT_NOTICE)).thenReturn(buildFeesData(feeForC2WithoutNotice));
         }
 
+        @Test
+        void shouldRetryPaymentsApiWhen500IsThrown() {
```

Comment:
500 is not throw, 500 is http status returned. maybe shouldRetryPaymentsApiWhenInternalServerErrorThrown

Reply:
done